UnitySDK/Assets/ML-Agents/Scripts/InferenceBrain/BarracudaModelParamLoader.cs

```diff
@@ -239,20 +239,8 @@ private static ModelActionType GetActionType(int isContinuousInt)
                     "The model does not contain a Vector Observation  Placeholder Input. " +
                     "You must set the Vector Observation Space Size to 0.");
             }
-            // If there are not enough Visual Observation Input compared to what the
-            // Brain Parameters expect.
-            for (var visObsIndex = 0;
-                 visObsIndex < brainParameters.cameraResolutions.Length;
-                 visObsIndex++)
-            {
-                if (!tensorsNames.Contains(
-                    TensorNames.VisualObservationPlaceholderPrefix + visObsIndex))
-                {
-                    failedModelChecks.Add(
-                        "The model does not contain a Visual Observation Placeholder Input " +
-                        "for visual observation " + visObsIndex + ".");
-                }
-            }
+
+            // TODO reenable checks there are enough Visual Observation Placeholder in the model.
```

Comment:
Do you have a way to do these checks ?

Reply:
➕

Reply:
Not currently, at least not at model load time, since we don't have any way to get the observation sizes until there's an Agent.